eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -68,6 +68,26 @@ jobs:
 
       - script: |
           node common/scripts/install-run-rush.js install
+        displayName: 'Install Rush'
```

Comment:
Let’s call this “Install dependencies” to match the other jobs

Reply:
done